### PR TITLE
Add encryption related so files to bin directory

### LINKED ISSUE #16: Linking error in 8.0.12-1
Version `8.0.12-1` has an implicit dependency on `libssl.so.1.0.0` and `libcrypto.so.1.0.0` being present in the user's environment.

While the packaging script adds these, it only adds them to the `lib` folder.  The standard MySQL distribution has these files present in both the `bin` and `lib` folders.  It appears that due to the way the binary is linked, it's not sufficient for these to be present in just the `lib` folder.

This can be verified by running the following in a Linux distribution.

```console
jar xf testing-mysql-server-8.0.12-1.jar
tar xvzf mysql-Linux-amd64.tar.gz
cd bin
ldd mysqld
```
Output:

```
[tdm@mymachine ~/mydir] ldd mysqld
	<<SNIP>>
	libssl.so.1.0.0 => not found
	libcrypto.so.1.0.0 => not found
	<<SNIP>>
```

If you compare this with the standard distribution of mysql, they get picked up because it picks up the local .so files in `bin`:

```
[tdm@mymachine ~/deleteme/mysql-8.0.15-linux-glibc2.12-x86_64/bin] ldd mysqld
	<<SNIP>>
	libssl.so.1.0.0 => /home/tdm/deleteme/mysql-8.0.15-linux-glibc2.12-x86_64/bin/./libssl.so.1.0.0 (0x00007fd1e1652000)
	libcrypto.so.1.0.0 => /home/tdm/deleteme/mysql-8.0.15-linux-glibc2.12-x86_64/bin/./libcrypto.so.1.0.0 (0x00007fd1e1218000)
	<<SNIP>>
```

Without the .so files the server fails to start:

```
/tmp/testing-mysql-server6111734346362237715/bin/mysqld: error while loading shared libraries: libssl.so.1.0.0: cannot open shared object file: No such file or directory

        at io.airlift.testing.mysql.EmbeddedMySql.system(EmbeddedMySql.java:280)
        at io.airlift.testing.mysql.EmbeddedMySql.initialize(EmbeddedMySql.java:158)
        at io.airlift.testing.mysql.EmbeddedMySql.<init>(EmbeddedMySql.java:80)
        at io.airlift.testing.mysql.TestingMySqlServer.<init>(TestingMySqlServer.java:54)
        at io.airlift.testing.mysql.TestingMySqlServer.<init>(TestingMySqlServer.java:44)
```

### PR DESCRIPTION
**Pull request by tdcmeehan**
Fixes #16

### CODE REVIEW

**Comment by tdcmeehan** [comment]
CC @electrum

File: repack-mysql.sh [hunk 1]
```diff
@@ -58,8 +58,8 @@ $TAR -czf $OLDPWD/$RESOURCES/mysql-Linux-amd64.tar.gz \
   share/*.txt \
   share/charsets \
   share/english \
-  lib/libcrypto.* \
-  lib/libssl.* \
+  bin/libcrypto.* \
```
Comment (electrum) [inline repack-mysql.sh]: This actually doesn't work, as these are symlinks in the original, and thus the produced tarball has dangling symlinks.

**Comment by electrum** [comment]
Nit: don't put "Fixes" in the commit description.

File: repack-mysql.sh (hunk 1 above)
Comment (tdcmeehan) [reply repack-mysql.sh]: Good catch, thanks!

**Comment by electrum** [comment]
Thanks for fixing this. I just released `8.0.12-2`.

**Pull request merged**